src/PaymentCalculator.java

```diff
@@ -1,12 +1,12 @@
 public class PaymentCalculator {
-    final double zus = 1000;
-    final double tax = 0.2;
+    private final static double ZUS = 1000;
```

Comment:
Nobody said to make them static, but that fine because they are not object dependent.